### PR TITLE
3rd pr neaz

### PR DESCRIPTION
**Pull request by Neaz-Mahmood**

### CODE REVIEW

**Review by ad3rinto — CHANGES_REQUESTED** [review]
You need to put the "|" at the end of your addition.

**Comment by Neaz-Mahmood** [comment]
resolved conflict issues

File: docs/TEAM.md [hunk 1]
```diff
@@ -23,8 +23,8 @@
...
 |[Titi Olopade](https://github.com/teckiegeek)| I am from the United Kingdom. My background is Computing and Statistics with a MSc. in Agile Project Management and PRINCE2 certified, SCRUM Master Certified. I have several years professional experience with few years career break. I started learning HTML and CSS coding 6months ago. I plan a change in career to be a Front-end developer. Here to learn and contribute to success of "twindle".|
-|[Ashish](https://github.com/kambleaa007) | 3+ yr of experience in .Net Java JavaScript Testing, Pune, India SoReady2Help :partying_face: |
-|[Nneyen ](https://github.com/Na-ilah) | I am from Nigeria. I am  beginner at github and open source projects. I have basic knowlege of Python, HTML and CSS. I don't remember much about programming even if I have a degree in computing. I would like to be able to contribute to the open source community and build educational solutions.|
+|[Neaz Mahmood](https://github.com/Neaz-Mahmood)|I am from Dhaka, Bangladesh. I am a second year Computer Science undergraduate. I know besic html, css, javascript. I am here to learn & contribute to 'twindle'. This is my first open source project.|
```
Comment (tolgaerdonmez) [inline docs/TEAM.md:26]: It would be nice to correct 'besic' to 'basic' :)

**Review by tolgaerdonmez — COMMENTED** [review]
Nice pr 👍

**Review by EYRAM565 — APPROVED** [review]

**Review by UnevenCoder — CHANGES_REQUESTED** [review]
Confluct again

**Review by EYRAM565 — APPROVED** [review]

**Pull request closed without merging**